pydicom/compat.py

```diff
@@ -1,34 +1,21 @@
 # Copyright 2008-2018 pydicom authors. See LICENSE file for details.
-"""Compatibility functions for python 2 vs later versions"""
+"""Compatibility functions for previous Python 2 support"""
```

Comment:
Maybe rename this to something like `utils`? Or would that introduce incompatibilities?